packages/html/src/listview/listview.spec.tsx

```diff
@@ -55,7 +55,7 @@ export const ListView = (
     const pager = <Pager className={`k-listview-pager k-listview-pager-${ pagerPosition }`} />;
 
     const loader = <>
-        <div className="k-loading-mask k-opaque" style={{ height: '100%', width: '100%' }}>
+        <div className="k-loading-mask" style={{ height: '100%', width: '100%' }}>
```

Comment:
Why are we removing the `k-opaque` class here?

Reply:
The `k-opaque` class makes the whole overlay with a solid white background. Therefore, nothing can be seen behind it.